tests/core/pyspec/eth2spec/test/phase0/fork_choice/test_ex_ante.py

```diff
@@ -0,0 +1,293 @@
+from eth2spec.test.context import (
+    MAINNET,
+    spec_configured_state_test,
+    spec_state_test,
+    with_all_phases,
+    with_presets,
+)
+from eth2spec.test.helpers.attestations import (
+    get_valid_attestation,
+    sign_attestation,
+)
+from eth2spec.test.helpers.block import (
+    build_empty_block,
+)
+from eth2spec.test.helpers.fork_choice import (
+    get_genesis_forkchoice_store_and_block,
+    on_tick_and_append_step,
+    add_attestation,
+    add_block,
+    tick_and_add_block,
+)
+from eth2spec.test.helpers.state import (
+    state_transition_and_sign_block,
+)
+
+
+def _apply_base_block_a(spec, state, store, test_steps):
+    # On receiving block A at slot `N`
+    block = build_empty_block(spec, state, slot=state.slot + 1)
+    signed_block_a = state_transition_and_sign_block(spec, state, block)
+    yield from tick_and_add_block(spec, store, signed_block_a, test_steps)
+    assert spec.get_head(store) == signed_block_a.message.hash_tree_root()
+
+
+@with_all_phases
+@spec_state_test
+def test_ex_ante_secnario_1_with_boost(spec, state):
```

Comment:
typo: `test_ex_ante_scenario_1_with_boost(spec, state):` not `test_ex_ante_secnario_1_with_boost(spec, state):`